jwst/datamodels/__init__.py

```diff
@@ -27,6 +27,7 @@
 from .ifucube import IFUCubeModel
 from .ifucubepars import NirspecIFUCubeParsModel, MiriIFUCubeParsModel
 from .ifuimage import IFUImageModel
+from .ifuptcor import MiriIFUPtCorrModel
```

Comment:
Do we know if this will only ever be used for MIRI MRS or is there a chance it could be used for NIRSpec IFU too? If it is MIRI-specific, then the module name should probably indicate that, e.g. "mrsptcor" or "miriptcor". Unlike modules such as photom or pathloss, this will likely only ever contain a single MIRI-specific datamodel.